main.go

```diff
@@ -0,0 +1,92 @@
+package main
+
+import (
+	"fmt"
+	"log"
+	"os"
+
+	"github.com/t-eckert/shepherd/cmd"
+	"github.com/t-eckert/shepherd/github"
+)
+
+func main() {
+	args := os.Args
+
+	command, err := cmd.NewCommand(args)
+	if err != nil {
+		log.Fatalln(err)
+	}
+
+	issues, err := github.FetchIssues(command.Origin)
+	if err != nil {
+		log.Fatalln(err)
+	}
+
+	for _, issue := range issues {
+		log.Printf("Issue #%d\t%s", issue.Number, issue.Title)
+	}
+
+	// Modify the prepend (e.g. helm:) if requested
+	if prepend, ok := command.Flags["modify-prepend"]; ok {
+		// Remove `:` at the end of prepend if user passed it in by accident
+		if prepend[len(prepend) - 1:] == ":" {
+			prepend = prepend[0:len(prepend) - 1]
+		}
```

Comment:
I think it's clearer to use `strings.HasSuffix` and `strings.TrimSuffix` here rather than indices.